frontend/simple/views/Contributions.vue

```diff
@@ -0,0 +1,195 @@
+<template>
+  <main>
+    <div class="section is-hero">
+      <i18n tag="h1" class="title is-1 is-marginless">
+        Contributions
+      </i18n>
+      <i18n tag="p">See everyone’s contributions to the group and manage yours.</i18n>
+    </div>
+
+    <div class="section columns is-desktop is-multiline c-grid">
+      <section class="column">
+        <i18n tag="h2" class="title is-3">Receiving</i18n>
+
+        <ul class="c-ul">
+          <contribution
+            v-for="contribution, index in receiving"
+            :variant="isMonetary(contribution.type) ? 'editable' : undefined"
+            v-on="isMonetary(contribution.type) ? { click: showReceivingSettings } : {}"
+          >
+            <strong v-if="isMonetary(contribution.type)">{{textGivingMonetary(index)}}</strong>
+            <strong v-else>{{contribution.what}}</strong>
+            <i18n>from</i18n>
+            {{getWho(contribution.who)}}
+            <template v-if="hasWhoElse(contribution.who)">
+              <i18n>and</i18n>
+              <tooltip>
+                <button class="gi-is-unstyled gi-is-link-inherit">{{contribution.who.length - 1}}<i18n>others</i18n></button>
+                <template slot="tooltip">
+                  <p v-for="name, index in contribution.who" v-if="index > 0">{{name}}</p>
+                </template>
+              </tooltip>
+            </template>
+          </contribution>
+        </ul>
+      </section>
+
+      <section class="column">
+        <i18n tag="h2" class="title is-3">Giving</i18n>
+
+        <ul class="c-ul" v-if="givesNonMonetary || givesMonetary">
+          <contribution
+            v-for="contribution, index in giving.nonMonetary"
+            class="has-text-weight-bold"
+            variant="editable"
+            @new-value="(value) => handleEditNonMonetary(value, index)"
+          >{{contribution}}</contribution>
+
+          <contribution v-if="givesMonetary"
+            variant="editable"
+            @click="showGivingMincomeSettings"
+          >
+            <span class="has-text-weight-bold">{{currency}}{{giving.monetary}}</span> <i18n>to other's mincome</i18n>
```

Comment:
Some languages may reverse the order of the information... so it might be better to place the `span` inside the `i18n`...